.omeroci/app-config

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+set -u
+
+# Call super. Activates virtualenv
+. /infra/app-config
+
+$OMERO_DIST/bin/omero config set omero.web.ui.top_links '[["Data", {"query_string": {"experimenter": -1}, "viewname": "webindex"}, {"title": "OMERO"}], ["Mapr", {"query_string": {"experimenter": -1}, "viewname": "maprindex_mapr"}, {"title": "Mapr browser"}]]'
```

Comment:
Do we want to include for the other apps the various configs e.g. open_with
I left it out for round 1

Reply:
Here I was starting to work towards a (more) usable installation a la: https://trello.com/c/2bBx96kX/23-mapr-installation

Certainly no objections from my side for us having more realistic configuration. I would like to get infra 0.3.0 in & tagged though so perhaps as a follow-up?

Reply:
that was more a design point. I will move it to the issue